lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -482,6 +483,10 @@ def configure(conf)
 
       @platform = detect_platform
 
+      # Unset the credential file path environment variable if the value is set
+      # to an empty string.
```

Comment:
Might want to explain why instead (e.g., `Treat an empty setting of the credentials file path environment variable as unset`).

Reply:
Changed. PTAL.